tsconfig.base.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
```

Comment:
I don't have powers to approve this. Would a start import work okay instead?

Reply:
I think I needed it to fix a TS error at one point, but I'll remove it for now.